### ISSUE TITLE
Export variance?

### ISSUE THREAD

**Issue by dswatson** [bgreenwell/fastshap#13]
Love the package! I'm wondering if there is some simple way to export the Monte Carlo variance for estimated phi values? This could help quantify uncertainty. Thanks!

**Comment by bgreenwell** [comment]
@dswatson, yes that seems like a reasonable thing to do. I’ll be sure to add this to the next release. Thanks for the suggestion!

**Comment by bgreenwell** [comment]
Possible now with `raw = TRUE`. See https://github.com/bgreenwell/fastshap/blob/main/NEWS.md#changed-1.
